src/geography.cpp

```diff
@@ -318,36 +345,92 @@ void init_geography(py::module &m) {
     pygeography.def(py::pickle([](Geography &geog) { return geog.encode(); },
                                [](py::tuple &encoded) { return Geography::decode(encoded); }));
 
+    py::options options;
+    options.disable_function_signatures();
+
     // Geography properties
 
     m.def("get_type_id",
           py::vectorize(&get_type_id),
           py::arg("geography"),
-          R"pbdoc(
-        Returns the type ID of a geography.
+          R"pbdoc(get_type_id(geography)
 
-        - None (missing) is -1
-        - POINT is 0
-        - LINESTRING is 1
+        Returns the type ID of a geography.
```

Comment:
I would personally keep this list here (but then expanded to be correct). Of course that gives two places we have to keep this up-to-date (although hopefully we won't change those values often after 0.1.0 ..). 
When using this function, it's nice to directly see what the return values mean without having to first check another docstring (when using the online documentation it is of course only a single click, but when in an interactive python session it is a bit more convoluted)

Or at least I would also mention ```:py:class:`GeographyType` ``` here already and not only in the returns section?